pkgs/applications/networking/cluster/habitat/default.nix

```diff
@@ -1,26 +1,26 @@
-{ stdenv, lib, fetchFromGitHub, rustPlatform, pkgconfig
-, libsodium, libarchive, openssl }:
+{ stdenv, lib, fetchFromGitHub, rustPlatform, /* pkgconfig
+, libsodium, libarchive, openssl */ }:
 
 with rustPlatform;
 
 buildRustPackage rec {
   name = "habitat-${version}";
-  version = "0.8.0";
+  version = "0.17.0";
 
   src = fetchFromGitHub {
     owner = "habitat-sh";
     repo = "habitat";
     rev = version;
-    sha256 = "1h9wv2v4hcv79jkkjf8j96lzxni9d51755zfflfr5s3ayaip7rzj";
+    sha256 = "0ff2410azr5a3cvqgbb35skv7ppah4hacfxlilpria8kzhp776gf";
   };
 
-  sourceRoot = "habitat-${version}-src/components/hab";
+  depsSha256 = "0sfnsxx2pbz59vg8x40s214v6cfg3wlff098fbdqii1i33qxkpaa";
 
-  depsSha256 = "1612jaw3zdrgrb56r755bb18l8szdmf1wi7p9lpv3d2gklqcb7l1";
+  buildInputs = [ /* libsodium libarchive openssl */ ];
 
-  buildInputs = [ libsodium libarchive openssl ];
+  nativeBuildInputs = [ /* pkgconfig */ ];
```

Comment:
Why are these commented out? https://github.com/habitat-sh/habitat/issues/994 isn't fixed, should -sh be retained? Is this PR ready to go?